www/js/app.js

```diff
@@ -354,7 +354,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         return false;
     });
     $('#btnHomeBottom').on('click', function() {
-        $('#btnHome').click();
+        document.getElementById("btnHome").click();
```

Comment:
It's minor, but could you please use the same quotation style as the surrounding code? I.e. in this case, use single quotation marks instead of double.